packages/web-ui/src/ds/atoms/Switch/index.tsx

```diff
@@ -86,7 +88,9 @@ function SwitchInput({
 
   return (
     <div
-      className={cn('flex flex-col gap-y-2 w-full', className)}
+      className={cn('flex flex-col gap-y-2', className, {
+        'w-full': fullWidth,
```

Comment:
you are changing the devault behaviour of all switch inputs

Reply:
No, `fullWidth = true`